File: ueueo-framework/ueueo-blob-storing/src/main/java/com/ueueo/BlobStoring/IBlobNormalizeNamingService.cs
namespace Volo.Abp.BlobStoring;

public interface IBlobNormalizeNamingService
{
    BlobNormalizeNaming NormalizeNaming(BlobContainerConfiguration configuration, String containerName, String blobName);

    String NormalizeContainerName(BlobContainerConfiguration configuration, String containerName);

    String NormalizeBlobName(BlobContainerConfiguration configuration, String blobName);
}

File: ueueo-framework/ueueo-blob-storing/src/main/java/com/ueueo/blobstoring/IBlobNormalizeNamingService.java
package com.ueueo.blobstoring;

public interface IBlobNormalizeNamingService {
    BlobNormalizeNaming normalizeNaming(BlobContainerConfiguration configuration, String containerName, String blobName);

    String normalizeContainerName(BlobContainerConfiguration configuration, String containerName);

    String normalizeBlobName(BlobContainerConfiguration configuration, String blobName);
}
